src/mpid/ch4/generic/am/mpidig_am_send.h

```diff
@@ -12,6 +12,13 @@
 #define MPIDIG_AM_SEND_FLAGS_SYNC (1)
 #define MPIDIG_AM_SEND_FLAGS_RTS (1 << 1)
 
+#define MPIDIG_AM_SEND_SET_RNDV(flags, id) \
```

Comment:
Need a comment showing the format (bit allocation) for the flags.

Reply:
Agreed. Will fix.

Reply:
Added.